elasticgraph-graphql/spec/unit/elastic_graph/graphql/datastore_query/search_index_expression_spec.rb

```diff
@@ -369,6 +369,24 @@ class GraphQL
 
               expect(parts).to target_all_widget_indices
             end
+
+            it "excludes no indices when we have an `any_of: [{anyof: []}]` filter because that will match all results" do
```

Comment:
Similar to my feedback about the shard routing: I think we can exclude all widget indices in this case because `{"any_of" => [{"any_of" => []}]}` matches no documents, right?